packages/client/src/debug/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as Debug } from "./Debug";
```

Comment:
Can you make sure all files in /debug are excluded from react test coverage via `istandbul`?

Reply:
check